src/charm.py

```diff
@@ -172,7 +282,13 @@ def _send_info(self, event, interfaces):
     def _check_leader(self):
```

Comment:
Add doc string. This code is not touched, but we should be bringing it up to standard.